src/components/MacAppStoreBadge/index.js

```diff
@@ -0,0 +1,57 @@
+import { Link, useTheme } from '@octopusthink/nautilus';
+import { css } from '@emotion/core';
+import React from 'react';
+
+import MacAppStoreBadgeSVG from 'src/images/mac-app-store-badge.svg';
+import { trackEvent } from '../../utils/eventTracking';
+
+const MacAppStoreBadge = () => {
+  const theme = useTheme();
+  const trackMASLink = () => {
+    trackEvent('MacAppStore-Click', 'Header');
+  };
+
+  return (
+    <div
+      css={css`
+        display: flex;
+        justify-content: center;
+      `}
+    >
+      <Link
+        as="a"
+        unstyled
+        css={css`
+          line-height: 0;
+          transition: all 100ms ease-in-out;
+
+          &:hover {
+            transform: scale(1.05);
+
+            img {
```

Comment:
This can be removed 😄

Reply:
Oh! Well, see. I'd like to actually SHOW the shadow, but something weird is happening where the image seems to .... acquire a background? I don't understand what's happening, because it's a transparent image, but I'd like to leave this for when/if I have a chance to figure out what's going on.

Reply:
Alright then, but add a comment explaining that 😏

Reply:
Done!